src/sentry/templates/sentry/vercel-ui-hook.vercel

```diff
@@ -0,0 +1,9 @@
+<Page>
+{% block main %}
+    <Box display="flex" justifyContent="space-between">
+      <P>
+        • <Link href="https://www.sentry.io/settings/{{ org }}/integrations/vercel/{{ integration_id }}">Add to project for {{ org }}</Link><BR />
```

Comment:
@ceorourke There should be a `/` after `{{ integration_id }}` since our URLs always end with one.